networks/suzuka/suzuka-full-node/src/main.rs

```diff
@@ -36,3 +32,15 @@ async fn run_suzuka() -> Result<ExitCode, anyhow::Error> {
 
 	Ok(ExitCode::SUCCESS)
 }
+
+fn init_tracing_subscriber() {
+	// TODO: compose console_subscriber as a layer
+	let env_filter = EnvFilter::builder()
+		.with_default_directive(LevelFilter::INFO.into())
+		.from_env_lossy();
+	let mut subscriber = tracing_subscriber::fmt().with_env_filter(env_filter);
```

Comment:
I think I'd like to have an option to write these spans directly to `.dat` perhaps by analysis. `aptos-core` has a setup where they use an enum to identify the table that their performance based logs should go into I believe. Probably easier to just use an generic type. 

But, here, even if it's just making the data mining a little be easier, it would be nice to have these written directly to a file.

Reply:
I need to look if it's possible to have a custom subscriber for this.

Reply:
Here's what I'm trying to implement quickly: a separate `fmt` subscriber with JSON output written to a file, logging only the closing events for any spans that match filtering directives. The directives will be configured from the value of the `SUZUKA_TIMING` environment variable.

JSON can then be massaged into any data format suitable for analysis.

Reply:
I've done something similar for the load and soak framework: https://github.com/movementlabsxyz/movement/blob/803354aa6c369885e0163b44001989f7729d6fa1/networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs#L27
Perhaps it can help.